index.html

```diff
@@ -58,16 +138,46 @@ <h1 class="logo">
     <section class="About-section">
       <div class="about">About</div>
       <p class="about-info">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
-        eiusmod
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
       </p>
-      <button type="button" class="cv">
+      <button type="button" class="connect">
         <img src="Group86.png" alt="yes" />
       </button>
```

Comment:
- [ ] One of the major requirements is that your design looks like that of the Figma template you choose. Currently, the button that says **_' Get my cv '_** is should cover its full width with little spacing to the side. currently, it covers like 3/4 of the space. Kindly make it identical to that of the Figma template you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/164946342-ba79847a-5d07-4b7b-9863-555dc6cc81b5.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/164946358-ddab8ace-5f02-4961-9e68-e82c1ddd8cc5.png)

- [ ] When I hover the button, I don't see the text **_' Get my cv '_** in its hover state. That is because you made it an image instead of a span with an icon. Kindly fi that also. 👍